config/environment.js

```diff
@@ -1,19 +0,0 @@
-'use strict';
-
-module.exports = function(environment /* , appConfig */) {
-  let ENV = {
-    environment,
-  };
-
-  if (environment === 'development') {
-    ENV['ember-a11y-testing'] = {
-      componentOptions: {
-        turnAuditOff: true, // Change to true to disable the audit in development
-        visualNoiseLevel: 2,
-        axeViolationClassNames: ['alert-box', 'alert-box--a11y'],
-      },
```

Comment:
Where does this configuration migrate to?

Reply:
The configuration was meant for the dummy app, so I'd guess `tests/dummy/config`. You'll want to check if `ember-a11y-testing` still support these options.

https://github.com/ember-a11y/ember-a11y-testing/tree/v7.1.2-ember-a11y-testing